SCons/Tool/JavaCommonTests.py

```diff
@@ -484,6 +484,41 @@ def test_nested_anonymous_inner_classes(self):
         expect = [ 'NestedExample$1', 'NestedExample$1$1', 'NestedExample' ]
         assert expect == classes, (expect, classes)
 
+    def test_lambda_after_new(self):
+        """Test lamdas after new"""
+
+        input = """\
+// import java.util.*;
+
+public class LamdaExample
+{
+
+    public void testFunc (int arg1, String arg2, Runnable lambda){
+    }
+    public LamdaExample()
+    {
+        testFunc(
+            5,
+            new String("test"),
+            () -> {}
```

Comment:
Stick a comment above the lambda to indicate that's what it is? (Not everyone's familiar with Java lambda syntax)

Reply:
updated in 1b2e8e9